core/codenvy-hosted-events/src/test/java/com/codenvy/user/interceptor/CreateUserInterceptorTest.java

```diff
@@ -55,6 +57,7 @@
     @BeforeMethod
     public void setup() throws Exception {
         setInterceptorPrivateFieldValue(interceptor, "apiEndpoint", "http://localhost/api");
+        setInterceptorPrivateFieldValue(interceptor, "mailSender", "noreplay@host");
```

Comment:
it's a test so any value can be set, but maybe you meant noreply instead of noreplay